README.md

```diff
@@ -15,6 +15,14 @@ The latest release includes the following versions of Istio and Envoy:
 **Istio version:** 1.21.3
 **Envoy version:** 1.29.5
 
+Istio Pilot Environment variables applied with `spec.compatibilityMode: true` on IstioCR:
```

Comment:
The main purpose of the `README` file is to explain what the Isito module is. For me, this information contains technical details. If you just want the users to know that there's a compatibility mode available when they look at the README, I'd use a link, for example: "If you want to enable compatibility with the previous minor version of Istio, see [Compatibility Mode](https://kyma-project.io/#/istio/user/00-10-overview-istio-controller?id=compatibility-mode). And then, you can add the table in the [Compatibility Mode](https://kyma-project.io/#/istio/user/00-10-overview-istio-controller?id=compatibility-mode) paragraph. I'm just afraid that if we continue with this approach, this file will just get cluttered with time.

Also, we have two README files in the Istio documentation: `/README` and `docs/user/README`. The README that [you can see on the website](https://kyma-project.io/#/istio/user/README) is pulled from the user folder. So, if you want this information to be displayed on the website, it should be duplicated.